pkg/oc/lib/describe/helpers.go

```diff
@@ -54,7 +55,16 @@ func bold(v interface{}) string {
 	return "\033[1m" + toString(v) + "\033[0m"
 }
 
-func convertEnv(env []api.EnvVar) map[string]string {
+// DEPRECATED:
+func convertEnvInternal(env []api.EnvVar) map[string]string {
```

Comment:
Is this still used by something? Do we need to keep it?

Reply:
not used, I can remove it in follow up if you're fine with that (don't want to mess with the queue now)